mkosi.md

```diff
@@ -1360,15 +1358,15 @@ Format=disk
 Packages=kernel,systemd,systemd-udev,openssh-clients,httpd
 BuildPackages=make,gcc,libcurl-devel
 EOF
-# cat >mkosi.build <<EOF
+$ cat >mkosi.build <<EOF
 #!/bin/sh
 cd $SRCDIR
 ./autogen.sh
 ./configure --prefix=/usr
 make -j `nproc`
 make install
 EOF
-# chmod +x mkosi.build
+$ chmod +x mkosi.build
 # mkosi --incremental boot
```

Comment:
That should also work as normal user, no?

Reply:
I thought that this uses systemd-nspawn which requires root?

Reply:
Exactly this still needs root

Reply:
My bad, when reading this I mixed it with the line above and read `build` instead of `boot`.